packages/gatsby-theme-docs/src/modules/ai-assistant/components/ai-assistant-launch-button.tsx

```diff
@@ -47,27 +58,17 @@ const AiAssistantLaunchButton = (props) => {
     />
   ) : (
     <AssistantLaunchContainer>
-      <IconButton
-        icon={<AssistantIcon />}
-        size="big"
-        label="Open AI assistant"
-        onClick={() => handleClick({ launch_location: 'topbar' })}
-        data-testid="ai-assistant-launch-button"
-      />
+      <Tooltip title="AI Assistant">
+        <IconButton
+          icon={<AssistantIcon />}
+          size="40"
```

Comment:
We had a depricated warning about size prop. I updated to the recommended number usage
<img width="1004" alt="Screenshot 2024-09-05 at 3 39 38 PM" src="https://github.com/user-attachments/assets/6a7953c0-a16d-4bed-8636-12ef42876c41">